core/services/flux_monitor_test.go

```diff
@@ -421,6 +471,7 @@ func TestOutsideDeviation(t *testing.T) {
 		{"inside deviation", decimal.NewFromInt(100), decimal.NewFromInt(101), 2, false},
 		{"equal to deviation", decimal.NewFromInt(100), decimal.NewFromInt(102), 2, true},
 		{"outside deviation", decimal.NewFromInt(100), decimal.NewFromInt(103), 2, true},
+		{"outside deviation zero", decimal.NewFromInt(100), decimal.NewFromInt(0), 2, true},
```

Comment:
This is perhaps outside the scope of this PR...

But how do you feel about adding test coverage for when threshold is negative?

Reply:
i feel validation and these constraints are outside the scope for sure.  It probably deserves its own story and we are waiting to hear back re: the audit.